modules/oe_content_publication/oe_content_publication.info.yml

```diff
@@ -5,8 +5,15 @@ package: OpenEuropa
 type: module
```

Comment:
oe_content_publication doesn't include README.md file. I think we should add it.

Reply:
I added the Readme.md file.